tensorflow/python/client/session_test.py

```diff
@@ -165,11 +165,30 @@ def testSessionInterOpThreadPool(self):
       results = s.run([inp], options=run_options)
       self.assertAllEqual([30.0], results)
 
-  def testErrorsReported(self):
-    with session.Session() as s:
-      constant_op.constant(10.0, name='W1')
-      with self.assertRaises(ValueError):
-        s.run('foo:0')
+  def testDeviceFinderTimeout(self):
```

Comment:
Why removing the original test case `testErrorsReported`?

Reply:
Deleted by mistake. Adding it back.